dashboards-observability/public/components/explorer/home.tsx

```diff
@@ -55,6 +56,12 @@ import { DeletePanelModal } from '../custom_panels/helpers/modal_containers';
 import { CUSTOM_PANELS_API_PREFIX } from '../../../common/constants/custom_panels';
 import { getSampleDataModal } from '../common/helpers/add_sample_modal';
 
+const pageStyles: CSS.Properties = {
+  float: 'left',
+  width: '100%',
+  maxWidth: '1130px',
+};
+
```

Comment:
I think this is already present in home.scss file. These lines might be redundant.

Reply:
You're right! Thanks!